install_config/upgrading/automated_upgrades.adoc

```diff
@@ -329,6 +329,91 @@ $ ansible-playbook -i <path/to/inventory/file> \
 See xref:../../admin_guide/manage_nodes.adoc#updating-labels-on-nodes[Manging
 Nodes] for more on node labels.
 
+
+[[upgrade-hooks]]
+=== Customizing Upgrades With Ansible Hooks
+
+When upgrading {product-title}, you can execute custom tasks during specific
+operations through a system called _hooks_. Hooks allow cluster administrators
```

Comment:
@sdodson FYI, avoided the awkward naming issue altogether here, since elsewhere in this topic we only refer to "installer" in the sense of the quick installer tool.